docs/ops/config.rst

```diff
@@ -179,6 +179,22 @@ in other Django projects.
 
     .. _mozlog: https://github.com/mozilla-services/Dockerflow/blob/master/docs/mozlog.md
 
+.. envvar:: DJANGO_CDN_URL
+
+  :default: ``None``
+
+  The URL of a CDN that is backed by Normandy, if one is in use. This is used to
+  enforce that immutable content is routed through the CDN. Must end with a
+  slash (``/``).
```

Comment:
Was this not documented before, or was I setting a configuration that wasn't in use before?